setup.cfg

```diff
@@ -1,4 +1,62 @@
 [metadata]
 name = detection_rules
+description = Detection Rules is the home for rules used by Elastic Security. This repository is used for the development, maintenance, testing, validation, and release of rules for Elastic Security’s Detection Engine.
+long_description = file: README.md
+version = 8.5.0
 license = Elastic License v2
 license_file = LICENSE.txt
+url = https://github.com/elastic/detection-rules
+classifiers =
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+    Programming Language :: Python
+    Topic :: Security
+    Topic :: Software Development :: Libraries :: Python Modules
+    Topic :: Software Development :: Libraries
+    Topic :: Software Development :: Testing
+    Topic :: Software Development
+    Topic :: Utilities
+[options]
+include_package_data = True
+packages =
+    detection_rules
+    shared
+    kql
+    kibana
+    rta
+install_requires =
+  Click~=8.1.0
+  elasticsearch~=8.1
+  eql==0.9.15
+  jsl==0.2.4
+  jsonschema==3.2.0
+  marko
+  marshmallow-dataclass[union]~=8.5.6
+  marshmallow-jsonschema~=0.12.0
+  marshmallow-union~=0.1.15
+  marshmallow~=3.13.0
+  pytoml
+  PyYAML~=5.3
+  requests~=2.27
+  toml==0.10.0
+  typing-inspect==0.7.1
+  XlsxWriter~=1.3.6
+python_requires = >=3.8.0
+
+[options.package_data]
+* = *.g
+
+[options.extras_require]
+dev =
+    pep8-naming==0.7.0
+    PyGithub==1.55
+    flake8==3.8.1
+    pyflakes==2.2.0
+    pytest>=3.6
+
+[options.entry_points]
+console_scripts =
+    detection_rules = detection_rules.__main__:main
```

Comment:
do we want (need) an entry point?

Reply:
It's neat, but TBH it doesn't full work because of all the hardcoded paths. So i'm going to sadly remove it for now.